src/Tabler.css

```diff
@@ -603,6 +603,12 @@ pre code {
   margin-left: auto;
 }
 
+.container.center {
```

Comment:
So that we can keep our new CSS separate from the base Tabler CSS can you put this into it's own new Container.css file and import it instead?